Automattic/simplenote-ios
# Nü Onboarding: Mark IV

### Details:
- **WPAuthHandler** is up for a refactor. Nevertheless, as discussed over Slack, since our deadlines are quite tight, we're using the old WordPress SSO code. [This method](https://github.com/Automattic/simplenote-ios/pull/386/files#diff-14403f026c26324c88a0e6af2f91fbeeR26) isn't new, and [comes from this spot](https://github.com/Automattic/simplenote-ios/blob/develop/Simplenote/Classes/SPLoginViewController.m#L160). We just moved stuff along!
- **SPSheetController** is a (super simple and not super generic) AlertController approach, that allows us to do a pixel-perfect approximation to the ActionSheet seen in the mockups
- New **UIColor**
- **UIKitConstants** is meant to help us avoid repeating the same stuff all over

Closes. #359

### Testing: Action Sheet
1. Fresh Install Simplenote
2. Press over the **LogIn** button

- [ ] Verify the ActionSheet looks great on iOS [13, 11, 10]
- [ ] Verify that pressing **Log in with Email** pushes the Authentication UI

### Testing: WordPress SSO:
1. Fresh Install Simplenote
2. Press over the **LogIn** button and pick the **WordPress SSO** option
3. Log into your WPCOM account

- [ ] Verify you're taken to your Simplenote account

### Testing: Error Handling
1. `git checkout 82a99ca`
2. Fresh install the app
3. Attempt a WordPress SSO Login

- [ ] Verify an alert shows up onscreen, indicating that something went off
